slim-iso/airootfs/root/customize_airootfs.sh

```diff
@@ -90,7 +90,7 @@ cd /usr/share/whatweb && rm -f Gemfile.lock &&
   bundle install --path vendor/bundle && rm -f Gemfile.lock
 
 # change default jdk
-archlinux-java set java-19-openjdk
+archlinux-java set java-20-openjdk
```

Comment:
That one doesn't exist, pick 21 or 24

https://wiki.archlinux.org/title/Java#OpenJDK